cloudinit/config/cc_mounts.py

```diff
@@ -367,180 +373,219 @@ def handle(name: str, cfg: Config, cloud: Cloud, args: list) -> None:
             if MNT_COMMENT in line:
                 fstab_removed.append(line)
                 continue
-
-            try:
-                toks = WS.split(line)
-            except Exception:
-                pass
+            toks = line.split()
             fstab_devs[toks[0]] = line
             fstab_lines.append(line)
-
-    device_aliases = cfg.get("device_aliases", {})
-
-    for i in range(len(cfgmnt)):
+    return fstab_lines, fstab_devs, fstab_removed
```

Comment:
I wondered about defining a namedtuple here for readability and usability of the return value from this function but I don't think it's worth the minor overhead here and we only have one call-site to keep aligned with this return value, so not a lot of reuse potential there.

Reply:
Agreed. Since it's just one site it doesn't seem like it would add much value.